UnitsNet.Tests/CustomCode/MassFlowTests.cs

```diff
@@ -48,6 +48,8 @@ public class MassFlowTests : MassFlowTestsBase
 
         protected override double TonnesPerDayInOneGramPerSecond => 60.0*60*24/1E6;
 
+        protected override double PoundsPerHourInOneGramPerSecond => 0.125998;
```

Comment:
Google says: `7.93664`
https://www.google.no/search?q=pounds+Per+Hour+In+1+Gram+Per+Second